pkg/azurefile/azurefile.go

```diff
@@ -886,6 +897,66 @@ func (d *Driver) ResizeFileShare(ctx context.Context, subsID, resourceGroup, acc
 	})
 }
 
+// CopyFileShare copies a fileshare in the same storage account
+func (d *Driver) copyFileShare(ctx context.Context, req *csi.CreateVolumeRequest, accountKey string, shareOptions *fileclient.ShareOptions, storageEndpointSuffix string) error {
+	if shareOptions.Protocol == storage.EnabledProtocolsNFS {
+		return fmt.Errorf("protocol nfs is not supported for volume cloning")
+	}
+	var sourceVolumeID string
+	if req.GetVolumeContentSource() != nil && req.GetVolumeContentSource().GetVolume() != nil {
+		sourceVolumeID = req.GetVolumeContentSource().GetVolume().GetVolumeId()
+	}
+	resourceGroupName, accountName, srcFileShareName, _, _, _, err := GetFileShareInfo(sourceVolumeID) //nolint:dogsled
+	dstFileShareName := shareOptions.Name
+	if err != nil {
+		return status.Error(codes.NotFound, err.Error())
+	}
+	klog.V(2).Infof("begin to copy fileshare %s to %s", srcFileShareName, dstFileShareName)
+	if srcFileShareName == "" {
```

Comment:
you could write like this, make the code shorter:
```
if if srcFileShareName == "" || dstFileShareName == "" {
   return fmt.Errorf("srcFileShareName(%s) or dstFileShareName(%s) is empty", srcFileShareName , dstFileShareName)
}

```

Reply:
done